src/pipe_wrap.h

```diff
@@ -21,6 +20,7 @@ class PipeWrap : public StreamWrap {
   size_t self_size() const override { return sizeof(*this); }
 
  private:
+  friend class ConnectionWrap;  // So handle_ can be accessed
```

Comment:
I'm just about to push a few commits, one of which changes the name back to handle_, but also one that removes this friend class as it was no longer required.